examples/test_field_sweep.py

```diff
@@ -0,0 +1,81 @@
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import numpy as np
+import matplotlib.pyplot as plt
+
+signal_pathway = {"ph1": 1}
+with psd.figlist_var() as fl:
+    for thisfile, exptype, nodename, post_proc, label_str in [
+        (
+            "240924_13p5mM_TEMPOL_field.h5",
+            "ODNP_NMR_comp/field_dependent",
+            "field_1",
+            "field_sweep_v4",
+            "9-24-24 13.5 mM TEMPOL sweep",
```

Comment:
I would argue this can/should be constructed from the date and chemical name -- something like
```
label_str = f"{s.get_prop('acq_params')['date']} {s.get_prop('acq_params')['chemical_name']} sweep"
```

Reply:
here "this" is the label

Reply:
I dont see how i can do this above where s is initialized